### PR TITLE
CHAD-4089 Remove legacy "status" variable from SmartSense MultiSensor

### PR DESCRIPTION
**Pull request by greens**

### CODE REVIEW

File: devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy [hunk 1]
```diff
@@ -73,15 +72,7 @@ metadata {
...
-		multiAttributeTile(name: "status", type: "generic", width: 6, height: 4) {
-			tileAttribute("device.status", key: "PRIMARY_CONTROL") {
-				attributeState "open", label: 'Open', icon: "st.contact.contact.open", backgroundColor: "#e86d13"
-				attributeState "closed", label: 'Closed', icon: "st.contact.contact.closed", backgroundColor: "#00a0dc"
-				attributeState "garage-open", label: 'Open', icon: "st.doors.garage.garage-open", backgroundColor: "#e86d13"
-				attributeState "garage-closed", label: 'Closed', icon: "st.doors.garage.garage-closed", backgroundColor: "#00a0dc"
-			}
-		}
-		standardTile("contact", "device.contact", width: 2, height: 2) {
+		standardTile("contact", "device.contact", width: 6, height: 4) {
```
Comment (tpmanley) [inline devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy]: Should still be a `multiAttributeTile` I think. Here is what we use for the `SmartSense Open/Close Sensor`:

```
multiAttributeTile(name: "contact", type: "generic", width: 6, height: 4) {
			tileAttribute("device.contact", key: "PRIMARY_CONTROL") {
				attributeState "open", label: '${name}', icon: "st.contact.contact.open", backgroundColor: "#e86d13"
				attributeState "closed", label: '${name}', icon: "st.contact.contact.closed", backgroundColor: "#00A0DC"
			}
		}
```

File: devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy [hunk 2]
```diff
@@ -271,7 +262,6 @@ private List<Map> translateZoneStatus(ZoneStatus zs) {
 		log.debug "Contact: ${device.displayName} value = ${value}"
 		def descriptionText = value == 'open' ? '{{ device.displayName }} was opened' : '{{ device.displayName }} was closed'
 		results << [name: 'contact', value: value, descriptionText: descriptionText, displayed: false, translatable: true]
```
Comment (tpmanley) [inline devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy]: remove `displayed` or set it to `true`

File: devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy [hunk 3]
```diff
@@ -353,18 +343,14 @@ List<Map> garageEvent(zValue) {
...
 	if (absValue > 900) {
 		contactValue = 'closed'
-		garageValue = 'garage-closed'
 	} else if (absValue < 100) {
 		contactValue = 'open'
-		garageValue = 'garage-open'
 	}
 	if (contactValue != null) {
 		def descriptionText = contactValue == 'open' ? '{{ device.displayName }} was opened' : '{{ device.displayName }} was closed'
 		results << [name: 'contact', value: contactValue, descriptionText: descriptionText, displayed: false, translatable: true]
```
Comment (tpmanley) [inline devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy]: `displayed` should be `true` or removed here too

**Comment by greens** [comment]
updated

**Review by tpmanley — APPROVED** [review]

**Pull request merged**